plugins/node/opentelemetry-instrumentation-koa/src/instrumentation.ts

```diff
@@ -126,7 +126,13 @@ export class KoaInstrumentation extends InstrumentationBase<typeof koa> {
     isRouter: boolean,
     layerPath?: string
   ): KoaMiddleware {
-    if (middlewareLayer[kLayerPatched] === true) return middlewareLayer;
+    const layerType = isRouter ? KoaLayerType.ROUTER : KoaLayerType.MIDDLEWARE;
+    // Skip patching layer if its ignored in the config
+    if (
+      middlewareLayer[kLayerPatched] === true ||
+      isLayerIgnored(layerType, this._config)
+    )
+      return middlewareLayer;
```

Comment:
With this if-statement on multiple lines I would prefer to use curly braces

```suggestion
    if (
      middlewareLayer[kLayerPatched] === true ||
      isLayerIgnored(layerType, this._config)
    ) {
      return middlewareLayer;
    }
```